torchvision/transforms/functional_pil.py

```diff
@@ -317,9 +317,9 @@ def rotate(
 @torch.jit.unused
 def perspective(
     img: Image.Image,
-    perspective_coeffs: float,
+    perspective_coeffs: List[float],
     interpolation: int = _pil_constants.BICUBIC,
-    fill: Optional[Union[float, List[float], Tuple[float, ...]]] = 0,
+    fill: Optional[Union[float, List[float], Tuple[float, ...]]] = None,
```

Comment:
Here is my understanding for why this is OK. The change on `fill` doesn't have any effect on the actual operation due to the call on `_parse_fill()`. Concerning the type of `perspective_coeffs` I believe that the change actually fixes a bug because PIL breaks if you pass a number:
```
TypeError: 'int' object is not subscriptable
```